### ISSUE TITLE
New Particle/Image feature issues.

### ISSUE THREAD

**Issue by Davros-and-Babble** [OpenCollarTeam/OpenCollar#102]
What version of OpenCollar are you using? OC 7 dev4

What behavior did you expect? 
1.  Hitting the back button on the image menu should bring you back to the previous page.  
2.  Expected that turning on an image would turn the title off.

What behavior did you see instead?  
1. Hitting the back button on the Image menu brings up the previous menu, but then opens the image menu again, undoing the back (so you end up on the same page).  
2. When selecting an image, the "title" stays on if it already was, and it places the title pretty far above the head.  You'll also notice that the default image height appears to be way higher than the default title height (when the image isn't being displayed):  https://gyazo.com/aa348bd8d1813410c0ca9721098ab038

What steps does someone need to take to reproduce the problem? 
1. Apps > Titler > Image > Back
2. Apps > Titler > Make sure "Show" is checked.   And then... Apps > Titler > Image > AFK (or some other image).

**Comment by ExecuterMagic** [comment]
Sadly dev 4 shows still almost the same issue with  an updated 3.x collar like before to me , when the image gets set , the  titler prim shoots far out to the front  ( so forward , before it also  moved downwards what it doesn't do now anymore as seems  ) , when the  image gets removed ( off ) again it returns to its old position .   

Apart  of that , i think it would be good to be able to set the height /  position of the titler and the image individual , what seems not  possible right now , so maybe should be split to 2 separate prims  instead of one ,to achieve that , and both prims be added by the updater  if not present before , like when one uses the colariser script to add  the OC features to unscripted items  ( i thought it to be less clutter  to add that here instead of in a new issue , if that was wrong i can  make a new one for it if needed ) .

**Comment by nirea** [comment]
Exe, unless you can provide me a collar that shows the misplaced prim issue, it's not going to change at this point.

I'll fix the back button issue.

Image is not meant to bump text.  You can set both at the same time.

**Comment by nirea** [comment]
OK, I figured out that the mispositioning of the titler was down to the collar using different attach points.  Attaching to chest would put the titler in front of you, while attaching to spine would put the titler behind you.  It turns out that each attach point has an inherent rotation that essentially changes which direction is "up".  I've just added corrections for chest and spine.  We can add others as needed, but I think that neck (which needs no correction) plus chest and spine should cover the vast majority of collars out there.
